test/test_xeus_python_kernel.py

```diff
@@ -0,0 +1,130 @@
+#############################################################################
+# Copyright (c) 2018, Martin Renou, Johan Mabille, Sylvain Corlay, and      # 
+# Wolf Vollprecht                                                           #
+# Copyright (c) 2018, QuantStack                                            #
+#                                                                           #
+# Distributed under the terms of the BSD 3-Clause License.                  #
+#                                                                           #
+# The full license is in the file LICENSE, distributed with this software.  #
+#############################################################################
+
+import tempfile
+import unittest
+import jupyter_kernel_test
+
+
+class XeusPythonTests(jupyter_kernel_test.KernelTests):
+
+    kernel_name = "xpython"
+    language_name = "python"
+
+    code_hello_world = "print('hello, world')"
+
+    code_page_something = "?print"
+
+    completion_samples = [
+        {'text': 'pri', 'matches': {'print'}},
+        {'text': 'from sys imp', 'matches': {'import'}},
+        {'text': 'se', 'matches': {'set', 'setattr'}},
+    ]
+
+    complete_code_samples = ['1', "print('hello, world')", "def f(x):\n  return x*2\n\n\n"]
+    incomplete_code_samples = ["print('''hello", "def f(x):\n  x*2"]
+    invalid_code_samples = ['import = 7q']
+
+    code_inspect_sample = "open"
+
+    def test_xeus_python_line_magic(self):
```

Comment:
We might want to remove all magics tests. We don't want our tests to break if IPython changes some magics logs etc.